util/qartod_service.py

```diff
@@ -74,28 +76,38 @@ def expand_qartod_record(record, subsite, node, sensor, stream, parameters):
         if record.stream is None:
             record.stream = stream
         # expand the null wildcard into all of the test records it represents
-        if record.parameter is None:
-            for parameter in parameters:
+        new_params = None
+        if record.parameters is None:
+            # parameters was null - add parameters to new records
```

Comment:
Maybe you could reword this comment. I'm thinking you mean "record.parameters was null - add parameters ..."